build.sbt

```diff
@@ -93,6 +94,15 @@ lazy val root = tlCrossRootProject
   )
   .aggregate(modules: _*)
 
+lazy val kernel = libraryCrossProject("kernel")
+  .settings(
+    startYear := Some(2024),
+    libraryDependencies := Seq(
+      catsCore.value,
+      catsEffectStd.value,
+    ),
+  )
```

Comment:
That module is currently blank on purpose, right?

Reply:
Oh, hell.  That shouldn't have been in this PR.